CAP/gap/DerivedMethods.gi

```diff
@@ -808,6 +808,28 @@ AddDerivationToCAP( IsIsomorphism,
     
 end : Description := "IsIsomorphism by deciding if it is a split mono and a split epi" );
 
+##
+AddDerivationToCAP( IsSplitEpimorphism,
+                    [ [ Lift, 1 ],
+                      [ IdentityMorphism, 1 ] ],
+  
+  function( morphism )
+    
+    return Lift( IdentityMorphism( Range( morphism ) ), morphism ) <> fail;
+  
+end : Description := "IsSplitEpimorphism by using lifts" );
+
+##
+AddDerivationToCAP( IsSplitMonomorphism,
+                    [ [ Colift, 1 ],
+                      [ IdentityMorphism, 1 ] ],
+  
+  function( morphism )
+    
+    return Colift( morphism, IdentityMorphism( Source( morphism ) ) ) <> fail;
```

Comment:
Here, please use `IsColiftable`

Reply:
The desired changes have been done in the 88ac5dc42c